apps/darts-modernisation/darts-automated-tasks/stg.yaml

```diff
@@ -13,3 +13,4 @@ spec:
         DARTS_LOG_LEVEL: DEBUG
         RESTART_ME: '007'
         AUTOMATED_TASK_MODE: true
+        API_POD: false
```

Comment:
Maybe way off the mark here so please ignore me but can we not use AUTOMATED_TASK_MODE

Reply:
You can here, but the integrated tests don't work as they expect both things to be true

Reply:
Can you make this change for all environments?